ui/apps/platform/src/Containers/NetworkGraph/namespace/NamespaceDeployments.tsx

```diff
@@ -23,8 +24,18 @@ const deployments = [
 ];
 
 function NamespaceDeployments() {
+    const [value, setValue] = React.useState('');
```

Comment:
nit: Use more descriptive var names
```suggestion
    const [searchValue, setSearchValue] = React.useState('');
```